salt/modules/rpm.py

```diff
@@ -168,14 +168,34 @@ def verify(*packages, **kwargs):
               'g': 'ghost',
               'l': 'license',
               'r': 'readme'}
+    options_map = {
+        'nocaps': '--nocaps',
+        'nodeps': '--nodeps',
+        'nodigest': '--nodigest',
+        'nofiledigest': '--nofiledigest',
+        'nofiles': '--nofiles',
+        'nogroup': '--nogroup',
+        'nolinkto': '--nolinkto',
+        'nomode': '--nomode',
+        'nomtime': '--nomtime',
+        'nordev': '--nordev',
+        'noscripts': '--noscripts',
+        'nosignature': '--nosignature',
+        'nosize': '--nosize',
+        'nouser': '--nouser'
+    }
     ret = {}
     ignore_types = kwargs.get('ignore_types', [])
+    verify_options = kwargs.get('verify_options', [])
+    cmd = ['rpm']
+    for option in verify_options:
+        cmd.append(options_map.get(option, ''))
```

Comment:
Yeah, it might not blow anything up, but it's really dependent on the command being run, since you will in essence be explicitly passing an empty string as an argument. I'd rather not leave it up to the whim of individual commands' tolerance of randomly-located empty string arguments, though.

Regardless, passing a bad option should at worst throw some sort of error, and silently ignoring it by doing a dictionary lookup that returns an empty string won't let it do that. My modifications to what you added allow for invalid arguments to cause the command to fail, letting the user investigate when the state fails to pass the package verification check. Though maybe I'm misremembering now that I type this... I think if the verification fails then we just reinstall the package, we'd want to raise an exception and then catch it and fail the state instead of just triggering a package reinstallation. I'll take a look at this.

I also noticed the ``ignore_types`` option and split it too in my fixes. Check the head of the develop branch.